README.md

```diff
@@ -213,6 +213,7 @@ docker run -p 80:8080 \
            -v /where/are/web/files:/www:ro \
            -e SERVER_NAME=app1.domain.com app2.domain.com \
            -e MULTISITE=yes \
+           -e ROOT_SITE_SUBFOLDER=app \   # optional subfolder level web root
```

Comment:
I think it will be better to do a dedicated section for ROOT_SITE_SUBFOLDER instead of editing the existing one.